ANSIBLE.md

```diff
@@ -0,0 +1,49 @@
+## Prerequisites
+* Ansible
+* Cloned Repository of GitHub Account
+
+## Quick start
+* Install ansible on your local machine.
+* Git clone this repository
+* Once the repository is cloned to your local machine, navigate to the ansible directory.
+* You will find a `playbook.yml` which is the base ansible script to run different roles(tasks) specified in it.
```

Comment:
The quick start section should have the bare minimum steps to push to production. Why does the reader need to know all this? (Ie roles and playbook etc). 

I'd suggest moving all ansible details to another heading. Keep quick start to 
1. Ssh keygen (because this is essential) 
2. Run ansible to staging (by default it should pull from the t4gc repo) 
3. Run ansible to prod 


Then you can add a section called "pulling code from your repo" where you can explain how to add the GitHub keys. 

The more content you add to the quick start section, the more possibilities for errors and confusion.